integration_tests/snapshots/correct-step-function-stack-snapshot.json

```diff
@@ -24,6 +24,10 @@
       "Key": "custom-tag-2",
       "Value": "tag-value-2"
      },
+     {
+      "Key": "DD_STEP_FUNCTIONS_TRACE_ENABLED",
```

Comment:
`DD_STEP_FUNCTIONS_TRACE_ENABLED `is used at the DD Forwarder, to enable Forwarder level trace enablement for SFs. For each individual SF, you need to use `DD_TRACE_ENABLED` instead of `DD_STEP_FUNCTIONS_TRACE_ENABLED`. In addition, we probably should only allow customer to enable `DD_STEP_FUNCTIONS_TRACE_ENABLED ` at the DD forwarder manually instead of via any ci/plugin/cdk tools when deploying an individual Step Functions.

Reply:
You are right. I set a wrong tag. Will fix.

Reply:
@nine5two7 Fixed!

Reply:
Later, we need to mention in the public documentation that this value is set to "true" by default. Customers can manually disable it in the AWS console. @lym953